src/handler_base.js

```diff
@@ -170,6 +170,15 @@ export default class HandlerBase extends EventEmitter {
         }
     }
 
+    getStatistics() {
```

Comment:
Maybe return null rather than undefined, so that users know which fields should be there but are missing.